source/RoslynAnalyzers/CaseSensitiveEnumParsingAnalyzer.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Octopus.RoslynAnalyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class CaseSensitiveEnumParsingAnalyzer : DiagnosticAnalyzer
+    {
+        const string DiagnosticId = "Octopus_CaseSensitiveEnumParsing";
```

Comment:
In general, do we prefer declaring const vars here for readability over the following:

 `        internal static readonly DiagnosticDescriptor Rule = new DiagnosticDescriptor(
            "Octopus_CaseSensitiveEnumParsing",
            "Enum.Parse is case sensitive",
            "The Enum.Parse call is case sensitive, which is likely not intended. Use an overload that explicitely defines case behaviour, or our `.ToEnum()` extension method.",
            "Octopus",
            DiagnosticSeverity.Error,
            true,
            @"Case sensitive Enum parsing has been the cause of several bugs. This analyzer forces the author to specify the casing behavior explicitely."
        );`

Reply:
It means that the values can be used from the unit tests if needed. It is also more readable I think.